testhw/helloworld/helloworld.bal

```diff
@@ -0,0 +1,33 @@
+import celleryio/cellery;
```

Comment:
This seems to be an unused file. Please remove

Reply:
Removed